content/en/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes.md

```diff
@@ -220,11 +220,53 @@ After 15 seconds, view Pod events to verify that liveness probes:
 kubectl describe pod goproxy
 ```
 
+## Define a gRPC liveness probe
+
+{{< feature-state for_k8s_version="v1.23" state="alpha" >}}
+
+If your application implements [gRPC Health Checking Protocol](https://github.com/grpc/grpc/blob/master/doc/health-checking.md),
+kubelet can be configured to use it for application liveness checks.
+
+{{< codenew file="pods/probe/content/en/examples/pods/probe/grpc-liveness.yaml">}}
+
+For the gRPC probe `port` have to be configured. If the health endpoint is configured
+on non-default service, `service` must be configured.
+
+{{< note >}}
+Unlike http and tcp probes, named ports cannot be used and custom host cannot be configured.
+{{< /note >}}
+
+Configuration problems - incorrect port and service, or not implemented health
+checking protocol are considered a probe failure similar to http and tcp probes.
+
+Before Kubernetes 1.23, gRPC health probes were often implemented using [grpc-health-probe](https://github.com/grpc-ecosystem/grpc-health-probe/),
+like described in the blog post [Health checking gRPC servers on Kubernetes](/blog/2018/10/01/health-checking-grpc-servers-on-kubernetes/).
+The built-in gRPC probes behavior is similar to one implemented by grpc-health-probe.
+When migrating from grpc-health-probe to built-in probes, remember the following differences:
+
+- built-in probes will run against pod IP, unlike grpc-health-probe that often runs against `127.0.0.1`.
+  Be sure to configure your gRPC endpoint to listen for pod IP address.
+- built-in probes do not currently support any authentication parameters (like `-tls`).
+- there is no error codes in built-in probes. All errors are considered a probe failures.
+
+In the example below, etcd pod is configured to use gRPC liveness probe.
+To try the gRPC liveness check, create a Pod:
```

Comment:
maybe flip these two sentences for clarity? e.g. "To try the gRPC liveness check, create a Pod. In this example, the etcd pod is configured to use the liveness probe."